AirCasting/CreateSessionViews/ChooseSessionTypeView.swift

```diff
@@ -101,7 +102,7 @@ struct ChooseSessionTypeView: View {
             _ = bluetoothManager.centralManager
             didTapFixedSession = true
         } else if CBCentralManager.authorization == .allowedAlways,
-           bluetoothManager.centralManager.state == .poweredOn {
+                  bluetoothManager.centralManager.state == .poweredOn {
```

Comment:
make sure it's not over-indented plz :)

Reply:
After clicking cmd+a and then ctr+I it goes to that state. I don't know what to think about it

Reply:
Personally I use ctrl+l only for the code I'm actually changing in a given PR. Doing cmd+a + ctrl+l can lead to unneeded conflicts and stuff like that. It's better to have a full control :)